cluster/examples/kubernetes/ceph/common.yaml

```diff
@@ -235,6 +237,8 @@ spec:
                 replicated:
                   properties:
                     size:
+                      maximum: 10
+                      minimum: 1
```

Comment:
@leseb looks good now?

Reply:
Yes. Once @travisn approves we will merge this.

Reply:
What about `cluster/examples/kubernetes/ceph/upgrade-from-v1.0-crds.yaml` when should I add a validation there, too?

Reply:
@sebastian-philipp validations should be added there if a PR is going to be backported to 1.1. At this point I'd suggest not backporting these changes or else we'll have to worry about upgrades for the users who have already upgraded.